### PR TITLE
add next page navigation to lp-page

### PR DESCRIPTION
**Pull request by HatiDev**
### JIRA link

https://companieshouse.atlassian.net/browse/LP-238

### Change description
add next page navigation to lp-page

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/presentation/controller/registration/Routing.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ const registrationRoutingGeneralPartners = {
 const registrationRoutingLimitedPartners = {
   previousUrl: WHICH_TYPE_URL,
   currentUrl: LIMITED_PARTNERS_URL,
-  nextUrl: NEXT_URL,
+  nextUrl: LIMITED_PARTNER_CHOICE_URL,
   pageType: RegistrationPageType.limitedPartners,
 };
```
Comment (mattch1) [inline src/presentation/controller/registration/Routing.ts:66]: Can we keep the page order here, so general partner choice above limited partners.

**Comment by mattch1** [comment]
Is the previous page for registrationRoutingLimitedPartnerChoice correct?

**Comment by mattch1** [comment]
Also related to the comment above look at the next page url for general partner choice AC1 says: ‘You now need to tell us about the limited partners’ page from ‘Is the general partner a person or a legal entity?’ 

so it looks like the flow generalPartners -> generalPartnerChoice -> limitedPartners -> LimitedPartnerChoice we should ensure that all previous and next urls in the routing reflect this.

**Comment by dlloyd1-ch** [comment]
This isn't working for me. It navigates from `Is the general partner a person or a legal entity?` straight to `Is the limited partner a person or a legal entity?` and misses out this page. 
What happens when you test locally?

**Review by markpit — APPROVED** [review]

**Review by lduranteau — APPROVED** [review]

**Review by mattch1 — APPROVED** [review]

**Pull request merged**